modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -11,6 +11,8 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 {productname} 7.0 was released for {enterpriseversion} and {cloudname} on Wednesday, March 20^th^, 2024. These release notes provide an overview of the changes for {productname} 7.0, including:
 
+* xref:new-premium-plugin<s>[New Premium plugins]
+* xref:new-open-source-plugin<s>[New Open Source plugin]
```

Comment:
@kemister85 I think we don't need this. As Markdown is now premium.

Reply:
Yes, this will be cleaned up in the final ticket for 7.0 release notes, which includes updating the `nav.adoc`